docbook/Admin_Guide/en-US/config/misc.xml

```diff
@@ -49,6 +49,15 @@
 			</listitem>
 		</varlistentry>
 		<varlistentry>
+			<term>$g_export_issues_threshold</term>
+			<listitem>
+				<para>Access level needed to export issues with Mantis
```

Comment:
Revise to "Access level required to export issues from MantisBT to formats like CSV and Excel.".